packages/cloud-cognitive/src/components/Datagrid/Datagrid/Datagrid.js

```diff
@@ -98,6 +98,7 @@ export let Datagrid = React.forwardRef(({ datagridState, ...rest }, ref) => {
       {leftPanel && (
         <div
           className={`${blockClass}__grid-container ${blockClass}__displayFlex`}
+          style={{ display: 'inherit' }}
```

Comment:
@faraznaseem019 

For the same reasons as above, this inline styling should definitely be declared in an `*.scss` unless if there is an exceptional case for having it here?